lib/sorcery/model/submodules/external.rb

```diff
@@ -96,14 +96,12 @@ def add_provider_to_user(provider, uid)
             authentications = sorcery_config.authentications_class.name.demodulize.underscore.pluralize
             # first check to see if user has a particular authentication already
             if sorcery_adapter.find_authentication_by_oauth_credentials(authentications, provider, uid).nil?
-              user = send(authentications).build(sorcery_config.provider_uid_attribute_name => uid,
+              authentication = send(authentications).build(sorcery_config.provider_uid_attribute_name => uid,
                                                  sorcery_config.provider_attribute_name => provider)
-              user.sorcery_adapter.save(validate: false)
+              authentication.sorcery_adapter.save(validate: false)
             else
-              user = false
+              false
```

Comment:
Actually, looking closer at this, I think this changes the return value when the authentication is present. Previously it would return `authentication`, but with this change it would instead return `authentication.sorcery_adapter.save(validate: false)` as that's the last value called in the method.

Can you revert the second part of this, and keep it to just renaming the variable to be more accurate?